modules/administration/pages/openscap.adoc

```diff
@@ -13,6 +13,14 @@ OpenSCAP auditing is not available on Salt SSH clients.
 To use openSCAP, you will need the [systemitem]``spacewalk-oscap`` package installed on the clients you want to audit.
 
 
+[IMPORTANT]
+====
+Scanning clients can consume a lot of memory on both the server and the client being scanned.
+You should ensure you have at least 2{nbsp}GB of RAM spare on each client to be scanned.
```

Comment:
No, we're okay with default-advised memory on SUSE clients (1 GB).

The problem is only for RedHat clients (where you need at least 1.25 GB according to my tests, but let's advice to use 2 GB).

Reply:
Ah, I misunderstood. Updated.